### ISSUE TITLE
Zoom suddenly is extremely slow

### ISSUE THREAD

**Issue by jupahe64** [exelix11/OdysseyEditor#2]
when I started this editor up everything was fine but suddenly the zoom was extremely slow, it looks like it has to do something with the inspect mode since it gets slower the more far you zoom in, but i'm in the walk around mode so it looks like this additional zoom factor doesn't get reset when switching the modes, which it SHOULD.

**Comment by exelix11** [comment]
Yes, this is a known issue of helixtoolkit, and also the reason because the walk around view is the default. The inspect mode uses the virtual trackball model, if you get too close to the target (aka zoom too much) the floating point operations will approximate to 0 and everything will start glitching, unfortunately the algorithm is beyond my control.
Solution: navigate with the middle mouse button and use a lot the space bar to move the camera target to the selected object.
Tbh once you get used to it i think the inspect mode is better
